.github/workflows/haskell.yml

```
@@ -62,22 +62,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.2.8", "9.6.6", "9.8.2", "9.10.1", "9.12.1"]
+        ghc: ["9.6", "9.8", "9.10", "9.12"]
         os: [ubuntu-latest, macos-13, macos-latest, windows-latest]
```

`macos-13` needs to be removed. It was only here for ghc-8.10
```suggestion
        os: [ubuntu-latest, macos-latest, windows-latest]
```